synapse/__init__.py

```diff
@@ -18,8 +18,8 @@
 if tornado.version_info < (3, 2, 2):
     raise Exception('synapse requires tornado >= 3.2.2')
 
-version = (0, 0, 26)
-verstring = '.'.join([str(x) for x in version])
+from synapse.lib.version import version
```

Comment:
Can you import both of these on one line?

Reply:
Technically, yes.